src/views/migrationpolicies/list/MigrationPoliciesList.tsx

```diff
@@ -54,16 +47,18 @@ const MigrationPoliciesList: FC = () => {
           loaded={loaded && loadedColumns}
           onFilterChange={onFilterChange}
         />
-        {loaded && isEmpty(mps) && <MigrationPoliciesEmptyState />}
-        <VirtualizedTable<V1alpha1MigrationPolicy>
-          columns={activeColumns}
-          data={data}
-          EmptyMsg={() => <></>}
-          loaded={loaded && loadedColumns}
-          loadError={loadError}
-          Row={MigrationPoliciesRow}
-          unfilteredData={unfilteredData}
-        />
+        {isEmpty(mps) && <MigrationPoliciesEmptyState loadError={!loaded && loadError} />}
+        {loaded && (
+          <VirtualizedTable<V1alpha1MigrationPolicy>
+            columns={activeColumns}
+            data={data}
+            EmptyMsg={() => <></>}
+            loaded={loaded && loadedColumns}
+            loadError={loadError}
+            Row={MigrationPoliciesRow}
+            unfilteredData={unfilteredData}
+          />
+        )}
```

Comment:
```suggestion
        <VirtualizedTable<V1alpha1MigrationPolicy>
          columns={activeColumns}
          data={data}
          EmptyMsg={() => <MigrationPoliciesEmptyState />}
          loaded={loaded && loadedColumns}
          loadError={loadError}
          Row={MigrationPoliciesRow}
          unfilteredData={unfilteredData}
        />
```

Reply:
This shouldn't keep loading and loading since the loaded is taking care of inside the VirtualizedTable component, and conditionally rendering according to loaded alone will miss if you have loadError as no permissions

Reply:
But we don't want to display the permission error in the UI anymore, it is in the design. Without the condition we cannot achieve the expected result. Or can we? Did you test it? How did it look?

Reply:
I saw the design and it's a mistake in my opinion.
Showing the list is about if a user have 'list' permission while create a new policy is checking for the 'create' permissions.
I'm discussing this with Yifat from UX, please keep following the Jira ticket, thanks :)